### PR TITLE
UI Smoke tests: updateSavedLoginTest & startBrowsingButtonTest

### PR DESCRIPTION
**Pull request by sv-ohorvath**
Added 2 new smoke tests, ran on Firebase 30 times and passed.

### CODE REVIEW

File: app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt [hunk 1]
```diff
@@ -32,6 +32,10 @@ import org.mozilla.fenix.helpers.ext.waitNotNull
 import org.mozilla.fenix.ui.robots.mDevice
 
 object TestHelper {
+    // Variable used to store currently running package name.
+    // Use this to replace hardcoded package name in UIAutomator matchers
+    val packageName = InstrumentationRegistry.getInstrumentation().targetContext.packageName
```
Comment (AaronMT) [inline app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt:37]: Is this used anywhere?

**Review by AaronMT — APPROVED** [review]

**Comment by AaronMT** [comment]
Not sure if the failure is related to the PR or not

File: app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt (hunk 1 above)
Comment (sv-ohorvath) [reply app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt:37]: oops, meant to, but it wasn't needed anymore. Thanks for catching this!

**Comment by sv-ohorvath** [comment]
> Not sure if the failure is related to the PR or not

@AaronMT It was related to this PR, should be fixed now.

**Pull request merged**